io.github.philipk.boilr.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Philip Kristoffersen 2022 <philipkristoffersen@gmail.com> -->
+<component type="desktop-application">
+  <id>io.github.philipk.boilr</id>
+  <metadata_license>CC0</metadata_license>
```

Comment:
this is an invalid SPDX identifier. See https://spdx.dev/licenses/

Reply:
The project is MIT or Apache, whatever the user wants.
For this I will just pick MIT-0 .
But just out of curiosity, is it possible to select multiple?

Reply:
`CC0` is not a valid identifier. In the list I linked it is `CC0-1.0`.

use `AND` or `OR` to combine multiple license. Identifiers from the list, again.

Reply:
I have moved this to upstream instead